eng/scripts/RunHelix.ps1

```diff
@@ -8,13 +8,12 @@
 .PARAMETER HelixQueues
     Set the Helix queues to use. The list is '+' or ';'-separated.
     Some supported queues:
-    Ubuntu.1604.Amd64.Open
     Ubuntu.1804.Amd64.Open
-    Windows.10.Amd64.Open
+    Ubuntu.2004.Amd64.Open
+    Windows.10.Amd64.Server20H2.Open
```

Comment:
`Windows.10.Amd64.Open` exists but we use `Windows.10.Amd64.Server20H2.Open` by default in real life. Matching that here.